app/GraphQL/Inventory/Builders/Products/ProductBuilder.php

```diff
@@ -17,6 +22,14 @@ public function getProducts(
         GraphQLContext $context,
         ResolveInfo $resolveInfo
     ): Builder {
+        $app = app(Apps::class);
+        $companyBranch = app(CompaniesBranches::class);
+        if ($companyBranch && key_exists('search', $args)) {
+            $region = Regions::getDefault($companyBranch->company);
+            $workflow = WorkflowStub::make(RainForestSearchWorkflow::class);
+            $workflow->start($app, auth()->user(), $companyBranch, $region, $args['search']);
```

Comment:
@FredPeal dont start a workflow like this , run a fireEvent for the entity tied to the workflow , also set this up as a feature flag you activate by app